doc/ipythonconsole.rst

```diff
@@ -27,14 +29,18 @@ Additionally, you can connect to external kernels managed by QtConsole sessions
 
 |
 
-When :guilabel:`Connect to an existing kernel` is selected, Spyder prompts for the connection details:
+When :guilabel:`Connect to an existing kernel` is selected, Spyder prompts for the connection information. If connecting to a remote kernel over ``ssh``, make sure to check the appropriate box and enter the required credentials in the listed fields.
 
 .. image:: images/console/console_dialog_connect.png
    :align: center
    :alt: Connect to kernel dialog, requesting path and connection details
 
 |
 
+For more technical details about connecting to remote ``IPython`` kernels, see the `Connecting to a remote kernel`_ page in the IPython Cookbook.
+
+.. _Connecting to a remote kernel: https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh
+
```

Comment:
This is not right. The idea is to inform people here how to connect to a remote kernel using Spyder.

For that they have to:

1. Start a kernel in the remote server with `ipython kernel`
2. Copy the file `kernel-<pid>.json` file or its contents to a local file.
3. Use the *Connect to an external kernel dialog* to enter the SSH connect info plus the local json file.

Reply:
The first two steps are outlined in the attached guide, and have to do with IPython rather than Spyder, while the third I do state in the docs. However, I'll mention them as you do above, clarify the third step and make clear the relevant portions of the linked guide for this use case. It isn't a number 1 priority right now, unlike the other three PRs here.

Reply:
Rewrote to be more clear/explicit about the steps instead of relying on the linked guide.

Reply:
Could you put the remote connection instructions in a separate subsection? Also, could you use a listing of instructions, as the ones I put above?

I think that would make easier for users to understand exactly what they need to connect to a remote kernel. Also, I think on Windows you need to install the `paramiko` to connect to Linux servers.

Reply:
Done and done, and added note about Windows. Also further clarified and explained the procedure, and fixed a minor consistency issue present in that section and elsewhere.